infrastructure-playbooks/rolling_update.yml

```diff
@@ -135,6 +135,12 @@
       set_fact:
         rolling_update: true
 
+    - name: ensure legacy ceph aliases file is absent
+      file:
+        path: /etc/profile.d/ceph-aliases.sh
```

Comment:
This is already done in the upgrade playbook since the beginning of the stable-4.0 branch and tags (look at first task in the mon play) so we shouldn't have any deployment (from scratch or upgraded) with that file anymore.

https://github.com/ceph/ceph-ansible/commit/57b4e76d11e0e3bb58c299ae5d37617ea06e2c17
https://github.com/ceph/ceph-ansible/pull/3718